.github/workflows/release.yaml

```diff
@@ -0,0 +1,49 @@
+name: Release
+on:
+  workflow_dispatch:
+  release:
+    types: [published]
+
+permissions:
+  contents: read
+
+jobs:
+  release:
+    permissions:
+      id-token: write
+      contents: read
+      attestations: write
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+
+    steps:
+
+    - name: Check out repository
+      uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+      with:
+        fetch-depth: 0
+
+    - name: Set up Apache Maven Central
+      uses: actions/setup-java@99b8673ff64fbf99d8d325f52d9a5bdedb8483e9 # v4.2.1
```

Comment:
Can this use `oracle-actions/setup-java` and pull in Java SE rather than temurin? We use that for the rest of our CI pipelines - https://github.com/oracle/tribuo/blob/main/.github/workflows/maven-ubuntu.yml#L23.

Reply:
Unfortunately, `oracle-actions/setup-java` doesn't seem to support the automatic deployment to Maven Central. We can, however, use the `oracle` distribution instead of `temurin`, which will use [the official Oracle JDK](https://www.oracle.com/java/technologies/downloads/).

Reply:
Alternatively, we can use `actions/setup-java` to handle credentials and `oracle-actions/setup-java` to compile the artifact. See commit [0d20fcb](https://github.com/oracle/tribuo/pull/373/commits/0d20fcba912d4ef8c2df411f9d1af52951b9483a).